openshift/templates/pims-app/deploy.yaml

```diff
@@ -41,37 +69,38 @@ objects:
         type: Rolling
       template:
         metadata:
-          name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+          name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
           labels:
-            name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+            name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
             app: ${APP_NAME}
             component: ${COMP_NAME}
             env: ${ENV_NAME}
             role: frontend
+            id: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
         spec:
           volumes:
-            - name: "${APP_NAME}-${COMP_NAME}-keycloak-config-volume"
+            - name: "${APP_NAME}-${COMP_NAME}${ID}-keycloak-config-volume"
               configMap:
-                name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}-sso-secret
+                name: keycloak-frontend-secrets
```

Comment:
no env vars here?

Reply:
Updated to be generic and support multiple instances.